app/components/Settings/scheduler/modals/create-modal/create-modal.jsx

```diff
@@ -109,29 +143,30 @@ class CreateModal extends Component {
                 query: this.state.code,
                 filename: generateFilename(),
                 cronInterval: this.state.interval,
-                name: this.state.name ? this.state.name.trim() : ''
+                name: this.state.name ? this.state.name.trim() : '',
+                tags: this.state.tags.map(t => t.id)
             })
-            .then((res) => {
+            .then(res => {
                 this.setState({
-                  successMessage: 'Scheduled query saved successfully!',
-                  saving: false,
-                  datasetUrl: getDatasetURL(res.fid)
+                    successMessage: 'Scheduled query saved successfully!',
```

Comment:
Just maybe fix spacing issues with previous lines

Reply:
let's ignore all spacing/formatting issues for now, as we'll run `prettier` on the entire codebase soon :)